pybandits/pydantic_version_compact.py

```diff
@@ -0,0 +1,242 @@
+"""
```

Comment:
Please add the MIT License on top of this file (as done in all other python files)